test/retry.test.js

```diff
@@ -148,17 +148,17 @@ describe("retry", function () {
                 retryBackoff: 2,
                 timeout: 30000
             });
-        })
+        });
         it("retryMaxDuration will surpass action timeout", function () {
             process.env.__OW_ACTION_DEADLINE = Date.now() + 2000;
             const options = retryInit();
             assertStartTime(options);
             assert.ok(options.retryMaxDuration < 3000, options.retryMaxDuration > 0);
-            assert.strictEqual(options.retryInitialDelay, 100);
-            assert.strictEqual(options.retryBackoff, 2);
-            assert.strictEqual(options.retryAllErrors, false);
-            assert.strictEqual(options.timeout, options.retryMaxDuration * 0.5);
-        })
+            assert.deepStrictEqual(options.retryInitialDelay, 100);
```

Comment:
This should stay as `strictEqual()`. The values compared to are all literal values, no objects. "deep" is only relevant for objects with nested structures.